src/static/js/views/faq.html

```diff
@@ -34,7 +37,10 @@ <h4>Frequently Asked Questions</h4>
         <p>More details of packages can be obtained directly from following links provided by respective distros.</p>
         <ul>
           <li><a href="https://www.debian.org/distrib/packages" target="blank">Debian 10 "buster"</a>. In this link, under the section <b>Search the contents of packages</b> select the Architecture as s390x from the dropdown to search for packages.</li>
-        </ul>
+		<li><a href="https://download.sinenomine.net/clefos/7/base/" target="blank">ClefOS 7</a>. In this link, go over the s390x directory</li>
+		<li><a href="https://download.opensuse.org/ports/zsystems/tumbleweed/" target="blank">OpenSUSE Tumbleweed</a>.</li>
+		<li><a href="https://download.opensuse.org/distribution/leap/15.3/repo/oss/" target="blank">OpenSUSE Leap</a>.</li>
+		</ul>
```

Comment:
So the thing about these links is that this is user-facing documentation, and these are not user-friendly. If someone is technical enough to want to know where the data is from, they will be able to look at your Python script directly.

The only links we want here are ones that are friendly to search, like the one for Debian. There isn't one for ClefOS, but we could link to https://software.opensuse.org/ for the SUSE distros.

Reply:
Sure, I'll do this

Reply:
What should we do about ClefOS then?

Reply:
Nothing :smile: we don't need a link to everything, just where there's something valuable to link to.